spec/counter_culture_spec.rb

```diff
@@ -98,6 +98,67 @@ def yaml_load(yaml)
     expect(product.reviews_count).to eq(1)
   end
 
+  it "skips increments counter cache on create" do
+    user = User.create
+    product = Product.create
+
+    expect(user.reviews_count).to eq(0)
+    expect(product.reviews_count).to eq(0)
+    expect(user.review_approvals_count).to eq(0)
+
+    Review.skip_counter_culture_updates do
+      user.reviews.create :user_id => user.id, :product_id => product.id, :approvals => 13
+    end
```

Comment:
Can you add some code below that makes sure the updates resume at the end of the block?